jyk2149.md

```diff
@@ -1,5 +1,7 @@
 # COMS4995
+License Badge: [![GitHub license](https://img.shields.io/github/license/jkim10/COMS-4995?style=for-the-badge)](https://github.com/jkim10/COMS-4995/blob/master/LICENSE)
```

Comment:
careful, GitHub renames don't last forever. You'll probably want to remap this from `COMS-4995` to `Spoonsor`.